spec.html

```diff
@@ -8360,21 +8391,20 @@ <h1>[[OwnPropertyKeys]] ( )</h1>
         </emu-alg>
       </emu-clause>
 
-      <emu-clause id="sec-integerindexedobjectcreate" aoid="IntegerIndexedObjectCreate">
-        <h1>IntegerIndexedObjectCreate ( _prototype_, _internalSlotsList_ )</h1>
-        <p>The abstract operation IntegerIndexedObjectCreate with arguments _prototype_ and _internalSlotsList_ is used to specify the creation of new <emu-xref href="#integer-indexed-exotic-object">Integer-Indexed exotic objects</emu-xref>. The argument _internalSlotsList_ is a List of the names of additional internal slots that must be defined as part of the object. IntegerIndexedObjectCreate performs the following steps:</p>
+      <emu-clause id="sec-IntegerIndexedObjectCreate" aoid="IntegerIndexedObjectCreate">
```

Comment:
Why change the clause id?

Reply:
If the change sticks, please add `oldids="sec-integerindexedobjectcreate"` to preserve preexisting links.

Reply:
Find-and-replace error.